controllers/contacts.js

```diff
@@ -0,0 +1,63 @@
+const { Contact } = require("../models/contact");
+const { ctrlWrapper, HttpError } = require("../helpers/index");
+
+const getAll = async (req, res) => {
```

Comment:
```suggestion
const getAllContacts = async (req, res) => {
```
Аналогічно для інших. Краще вказувати більш явну назву